src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -393,31 +398,35 @@ def exposure_effect(rates, rr: pd.DataFrame) -> pd.Series:
 
 
 def get_population_attributable_fraction_data(
-    builder, risk: EntityString, target: TargetString
-):
-    rr_source_type = validate_relative_risk_data_source(builder, risk, target)
-    exposure_source = builder.configuration[risk]["data_sources"]["exposure"]
-    if (
-        is_data_from_artifact(exposure_source)
-        and rr_source_type == "data"
-        and risk.type == "risk_factor"
-    ):
-        paf_data = builder.data.load(
-            builder.configuration[risk]["population_attributable_fraction"]["key_name"]
-        )
+    builder: Builder, risk: EntityString, target: TargetString
+) -> Tuple[pd.DataFrame, List[str]]:
+    paf_data = builder.data.load(f"{risk}.population_attributable_fraction")
+    if isinstance(paf_data, pd.DataFrame):
         correct_target = (paf_data["affected_entity"] == target.name) & (
             paf_data["affected_measure"] == target.measure
         )
         paf_data = paf_data[correct_target].drop(
             columns=["affected_entity", "affected_measure"]
         )
     else:
-        key_cols = ["sex", "age_start", "age_end", "year_start", "year_end"]
-        exposure_data = get_exposure_data(builder, risk).set_index(key_cols)
-        relative_risk_data = get_relative_risk_data(builder, risk, target).set_index(key_cols)
+        exposure_data, exposure_value_cols = get_exposure_data(builder, risk)
+        relative_risk_data, rr_value_cols = get_relative_risk_data(builder, risk, target)
```

Comment:
nit: maybe wrap `exposure_val_cols` and `rr_value_cols` in `set` up front so you don't have to do it every time.